asset-store/resources/pages-bucket.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: assetstore.kyma-project.io/v1alpha2
```

Comment:
What is the purpose of these resources as you don't mention them in documentation?